frontend/src/components/admin.js

```diff
@@ -140,10 +143,13 @@ function ReportCard() {
   const [respText, setRespText] = useState([]);
   const handleClick = () => setLoading(true);
   const jwt = JSON.parse(localStorage.getItem('jwt'));
-  const data = {
-      jwt: jwt["jwt"],
-  }
   useEffect(() => {
+    if (jwt == null) {
+      return;
+    }
+    const data = {
+      jwt: jwt["jwt"],
+    }
```

Comment:
This fixes the bug that Justin Silver found?

Reply:
Yup that's the one. I discovered it when testing room swaps using incognito windows 😑